lib/setupHooks/installFromCargoBuildLogHook.sh

```diff
@@ -13,12 +13,18 @@ function installFromCargoBuildLog() (
   logs=$(jq -R 'fromjson?' <"${log}")
 
   local select_non_test='select(.reason == "compiler-artifact" and .profile.test == false)'
-  local select_bins="${select_non_test} | .executable | select(.!= null)"
+  local select_non_dep='select((contains("/release/deps/") or contains("/debug/deps/"))| not)'
```

Comment:
Should this check be shortened to just `select(contains("/deps/") | not)`? `release` and `dev` aren't the only profiles and additional custom profiles could trip this up

(Also, is there any other metadata that cargo emits in the json output to denote this as a build dep or is the path the only way to check for it?)

Reply:
I did not manage to find anything else than that, which is why I added the profile, just to make sure we're not grabbing something we should not (or we're in some weird edge case with e.g. nix store path set to `/deps` or something like that)

I'll remove the profile